pages/start.tsx

```diff
@@ -153,16 +160,36 @@ const LoginStep2: React.FC<LoginStep2Props> = ({ handleLoginStep, setRepoList })
         });
 
         setRepoList(repoList);
+        setAddOrgError(OrgLookupError.Initial);
       }
 
       // If valid, go to next step
       handleLoginStep();
-    } catch (e) {
+    } catch (e: any) {
       // If invalid, display error
       console.error(e);
+      
+      if ( e.code  == 404) {
+        setAddOrgError(OrgLookupError.Invalid);
+      } 
+      else {
+        setAddOrgError(OrgLookupError.Error);
+      } 
     }
   };
 
+  const getOrgLookupError = (code: OrgLookupError) => {
```

Comment:
this code split is not re-usable, the actual error message is not passed around; please apply the conditionals in the display view in a way that the user can read the error message and be alerted of the error.

Reply:
This is the same logic used for the 'Repo Look-up error' on the third step of the onboarding. So I just tried to keep the consistency of the code structure. And this actually re-useable and helps logically show the error message to the user based on the HTTP response code. For example only if the response code is 404 it will show the message "The organization entered is invalid." otherwise it will show "There was an error retrieving this organization. I think these two types of error and the corresponding error message are enough to get a user to understand what's gone wrong. Also as it is reusable we can easily add a single line of code to add another type of error message.

Reply:
both the org lookup and repo lookup can be improved; can you improve the part that is related to this pr?

Reply:
Let's start by making error messages a hue or red. These pills have a hue for the background as well.